dipper/sources/BioGrid.py

```diff
@@ -205,6 +207,32 @@ def _get_interactions(self, limit):
 
         return
 
+    def _interactor_to_gene_curie(self, interactor: str) -> Union[str, None]:
```

Comment:
Check out Optional - https://docs.python.org/3/library/typing.html#typing.Optional  Same thing but perhaps more readable? it's subjective

Reply:
Okay, done - thanks